docs/augs_overview/image_only/image_only.rst

```diff
@@ -29,3 +29,171 @@ API link: :class:`~albumentations.augmentations.transforms.ToSepia`
 
 .. figure:: ./images/ToSepia.jpg
     :alt: ToSepia image
+
+JpegCompression
+-------------------
+
+
+API link: :class:`~albumentations.augmentations.transforms.JpegCompression`
+
+
+1. Original image
+2. :code:`JpegCompression(quality_lower=99, quality_upper=100, p=1.0)`
+3. :code:`JpegCompression(quality_lower=59, quality_upper=60, p=1.0)`
+4. :code:`JpegCompression(quality_lower=39, quality_upper=40, p=1.0)`
+5. :code:`JpegCompression(quality_lower=19, quality_upper=20, p=1.0)`
+6. :code:`JpegCompression(quality_lower=0, quality_upper=1, p=1.0)`
+
+.. figure:: ./images/JpegCompression.jpg
+    :alt: JpegCompression image
+
+Blur
+-------------------
+
+
+API link: :class:`~albumentations.augmentations.transforms.Blur`
+
+
+1. Original image
+2. :code:`Blur(blur_limit=7, p=1.0)`
+3. :code:`Blur(blur_limit=15, p=1.0)`
+4. :code:`Blur(blur_limit=50, p=1.0)`
```

Comment:
i don't think blur with kernel >~21 makes any sence

Reply:
It depends on the use case. I would say if the kernel is smaller than image => good enough.

Reply:
Do you know how blur works?